_events/2018-05-18-de.STAIR-primer-rna-seq.md

```diff
@@ -1,5 +1,5 @@
 ---
-site: main
+site: freiburg
```

Comment:
Does it mean that we do not have anymore general news for Europe? 
Because this event for example is not our event: it is from de.STAIR